tests/conftest.py

```diff
@@ -13,7 +13,7 @@ def tls_certificate_authority() -> trustme.CA:
 
 @pytest.fixture
 def tls_certificate(tls_certificate_authority: trustme.CA) -> trustme.LeafCert:
-    return tls_certificate_authority.issue_server_cert(
```

Comment:
this is equivalent isn't it, why chage it ?

Reply:
https://github.com/python-trio/trustme/blob/f9e13e03963260b32cab03f4e4c25cc515ddfad3/docs/source/index.rst#trustme-050-2019-01-21 it's a legacy name for compatibility